devices/tuya.js

```diff
@@ -358,6 +358,14 @@ module.exports = [
         fromZigbee: [fz.on_off, fz.tuya_led_controller, fz.brightness, fz.ignore_basic_report],
         exposes: [e.light_brightness_colortemp_colorhs().removeFeature('color_temp_startup')],
     },
+    {
+        fingerprint: [{modelID: 'TS0505A', manufacturerName: '_TZ3000_keabpigv'}],
+        model: 'R9077',
+        vendor: 'Woox',
```

Comment:
Since this is a different vendor, please create a new file `woox.js` instead of adding it to `tuya.js`.

Reply:
Done. I've also moved the other Woox device from `tuya.js` to the new file.

Reply:
Thanks!